embedly/views.py

```diff
@@ -1,15 +1,17 @@
 """Embed.ly views"""
 from urllib.parse import unquote
 from rest_framework import status
-from rest_framework.decorators import api_view
+from rest_framework.decorators import api_view, permission_classes
 from rest_framework.response import Response
 
 from django.conf import settings
 
 from embedly.api import get_embedly
+from open_discussions.permissions import AnonymousAccessReadonlyPermission
 
 
 @api_view()
+@permission_classes([AnonymousAccessReadonlyPermission])
```

Comment:
Could you add a test case for anonymous users to be sure we can catch if this breaks?